### ISSUE TITLE
ModuleNotFoundError: No module named 'niworkflows.interfaces.registration'

### ISSUE THREAD

**Issue by narloncassio** [nipreps/mriqc#934]
Hi there,
I am trying to use mriqc but I'm running into the following problem. 

I would appreciate your help!

$ docker run -it --rm -v /Users/Narlon/Desktop/mriqc_test:/data:ro -v /Users/Narlon/Desktop/mriqc_test:/out poldracklab/mriqc:latest /data /out participant --participant_label 001
210924-03:17:45,873 cli IMPORTANT:
	 
    Running MRIQC version 0.16.1+81.g0093e62.dirty:
      * BIDS dataset path: /data.
      * Output folder: /out.
      * Analysis levels: ['participant'].

Process Process-2:
Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/conda/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/conda/lib/python3.8/site-packages/mriqc/cli/workflow.py", line 21, in build_workflow
    retval["workflow"] = init_mriqc_wf()
  File "/opt/conda/lib/python3.8/site-packages/mriqc/workflows/core.py", line 28, in init_mriqc_wf
    workflow.add_nodes([anat_qc_workflow()])
  File "/opt/conda/lib/python3.8/site-packages/mriqc/workflows/anatomical.py", line 82, in anat_qc_workflow
    norm = spatial_normalization()
  File "/opt/conda/lib/python3.8/site-packages/mriqc/workflows/anatomical.py", line 165, in spatial_normalization
    from niworkflows.interfaces.registration import \
ModuleNotFoundError: No module named 'niworkflows.interfaces.registration'

**Comment by nwd2918** [comment]
I'm getting a similar error as follows: 

```
Running MRIQC version 0.16.1+81.g0093e62.dirty:
      * BIDS dataset path: /data.
      * Output folder: /out.
      * Analysis levels: ['participant'].

Process Process-2:
Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/conda/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/conda/lib/python3.8/site-packages/mriqc/cli/workflow.py", line 21, in build_workflow
    retval["workflow"] = init_mriqc_wf()
  File "/opt/conda/lib/python3.8/site-packages/mriqc/workflows/core.py", line 22, in init_mriqc_wf
    workflow.add_nodes([fmri_qc_workflow()])
  File "/opt/conda/lib/python3.8/site-packages/mriqc/workflows/functional.py", line 43, in fmri_qc_workflow
    from niworkflows.interfaces.utils import SanitizeImage
ModuleNotFoundError: No module named 'niworkflows.interfaces.utils'
```

The same problem occurs on an Ubuntu 20.04 system and on a macOS (v10) system.

**Comment by oesteban** [comment]
Do you get the same issue using the new docker image? i.e. `nipreps/mriqc:latest` instead of `poldracklab/mriqc:latest`

**Comment by nwd2918** [comment]
Yes, the same error occurs:

```
latest: Pulling from nipreps/mriqc
Digest: sha256:e4b0e2147368f78f74841a2139bb609f6fdaccc25514479688fca629bdf2aa74
Status: Downloaded newer image for nipreps/mriqc:latest
210924-10:00:31,964 cli IMPORTANT:
	 
    Running MRIQC version 0.16.1+81.g0093e62.dirty:
      * BIDS dataset path: /data.
      * Output folder: /out.
      * Analysis levels: ['participant'].

Process Process-2:
Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/conda/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/conda/lib/python3.8/site-packages/mriqc/cli/workflow.py", line 21, in build_workflow
    retval["workflow"] = init_mriqc_wf()
  File "/opt/conda/lib/python3.8/site-packages/mriqc/workflows/core.py", line 22, in init_mriqc_wf
    workflow.add_nodes([fmri_qc_workflow()])
  File "/opt/conda/lib/python3.8/site-packages/mriqc/workflows/functional.py", line 43, in fmri_qc_workflow
    from niworkflows.interfaces.utils import SanitizeImage
ModuleNotFoundError: No module named 'niworkflows.interfaces.utils'
```

**Comment by narloncassio** [comment]
Same for me, the error persists.

**Comment by oesteban** [comment]
Okay, I just checked and this has been fixed with `nipreps/mriqc:21.0.0rc1`. Please reopen if this problem regresses.
